pkg/apis/config/features_test.go

```diff
@@ -140,6 +142,22 @@ func TestFeaturesConfiguration(t *testing.T) {
 		data: map[string]string{
 			"responsive-revision-gc": "Enabled",
 		},
+	}, {
+		name:    "prevent-active-revision-deletion Allowed",
+		wantErr: false,
+		wantFeatures: defaultWith(&Features{
+			PreventActiveRevisionDeletion: Allowed,
+		}),
+		data: map[string]string{"prevent-active-revision-deletion": "Allowed"},
```

Comment:
nit (just to match rest of the table):
```suggestion
		data: map[string]string{
			"prevent-active-revision-deletion": "Allowed",
		},
```

Reply:
Fixed.